### PR TITLE
Updates for EN, ES, FR

### PR DESCRIPTION
**Pull request by ignotus666**
# Does this need translation?

- [ ] Yes 
- [x] No 

# Changes

--centralserver changed to --directoryserver in EN, update to ES docs, fixed anchors in FR. The mistakes in the first commit have been fixed in the second one (outdated translation memory in OmegaT - fixed).

The {:.no_toc} stuff keeps coming out different due to po4a (I'm using it to create all this stuff) but it looks fine in the resulting document.

### CODE REVIEW

**Comment by ignotus666** [comment]
Arrrggh... I think that last commit (fix formatting) reverted corrections to DE and PT... It was only meant to fix the 
{: .button}\\ **Mirror 2:** formatting but all that other stuff got in. I don't know how to discard that last commit :(

**Comment by jujudusud** [comment]
https://gist.github.com/gunjanpatel/18f9e4d1eb609597c50c2118e416e6a6

**Comment by ignotus666** [comment]
Thanks, I'll have s look at that later. Or maybe I'll just fix those docs as I'll have to do it sooner or later anyway.

File: wiki/de/de-Installation-for-Windows.md [hunk 1]
```diff
@@ -1,54 +1,56 @@
...
 
 ***
 
 ## ASIO
 
 Jamulus nutzt [ASIO](https://de.wikipedia.org/wiki/Audio_Stream_Input/Output), um mit der möglicht geringsten Audiolatenz zu arbeiten.
-
 * Wenn du eine externe Soundkarte bzw. -interface besitzt, nutze den offiziellen ASIO-Treiber (normalerweise bieten sie die beste Qualität).
-* Wenn du keine externe Soundkarte besitzt, wirst du wahrscheinlich keinen ASIO-Treiber haben. Deshalb musst du einen generischen Treiber nutzen, wie ASIO4ALL:
+* Wenn du keine externe Soundkarte besitzt, wird es wahrscheinlich keinen ASIO-Treiber haben. Deshalb musst du einen generischen Treiber nutzen, wie ASIO4ALL:
```
Comment (ann0see) [inline wiki/de/de-Installation-for-Windows.md]: Didn't we change this one @hoffie ? Grammar wise this sounds strange to me.

File: wiki/es/es-Command-Line-Options.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Por ejemplo, en Windows, para que un cliente utilice un archivo específico de c
 
 Para macOS, arranca una ventana de Terminal y ejecuta Jamulus con las opciones deseadas como sigue:
 
-/Applications/Jamulus.app/Contents/MacOS/Jamulus --inifile "/ruta/a/myinifile.ini"
+```shell /Applications/Jamulus.app/Contents/MacOS/Jamulus --inifile "/path/to/myinifile.ini" ```
```
Comment (ann0see) [inline wiki/es/es-Command-Line-Options.md]: Long commands could cause an overflow on mobile devices. We might need to use:

```suggestion
```shell
/Applications/Jamulus.app/Contents/MacOS/Jamulus --inifile "/path/to/myinifile.ini"
```
```
which generates a scroll-able code box

File: wiki/es/es-Software-Manual.md [hunk 3]
```diff
@@ -4,18 +4,15 @@ layout: wiki
 permalink: /wiki/Software-Manual
 title: 'Manual del Software'
 ---
-# Manual de Usuario de Jamulus
-
-{:.no_toc}
+# Manual de Usuario de Jamulus {:.no_toc}
```
Comment (ann0see) [inline wiki/es/es-Software-Manual.md]: I hope we don't need to fix this manually every time.

"toc" seems to be "Table of contents"

File: wiki/de/de-Installation-for-Windows.md (hunk 1 above)
Comment (hoffie) [reply wiki/de/de-Installation-for-Windows.md]: Don't remember exactly, but the new version sounds wrong and the old version sounds acceptable to me (but maybe we even had another alternative? Not sure)

Comment (ignotus666) [reply wiki/de/de-Installation-for-Windows.md]: Don't pay attention to the changes in German - they crept in due to a mistake. I'll fix them tonight.

File: wiki/es/es-Software-Manual.md (hunk 3 above)
Comment (ignotus666) [reply wiki/es/es-Software-Manual.md]: I formatted the version in EN so the processed version with po4a comes out correctly. It shouldn't need editing again.

Comment (ann0see) [reply wiki/es/es-Software-Manual.md]: At least for ES it produces a wrong html file: 

toc

Comment (ignotus666) [reply wiki/es/es-Software-Manual.md]: Fixed it - hopefully. The EN version wasn't formatted properly. It's working ok now locally. This PR still needs some work to fix the mess in German though. I hope to get to it tomorrow.

**Comment by ignotus666** [comment]
I think I've corrected all the DE and PT mistakes now - they should be up to date with the 'translate' version with a couple of corrections. Checked the site built locally and it all looks good.

**Review by ann0see — COMMENTED** [review]
Thank you!

**Comment by jujudusud** [comment]
Are the screenshots ok ?

**Comment by ignotus666** [comment]
There's one missing for DE (settings-profile), all of them missing for PT, and FR and ES have them all working ok.

**Comment by ann0see** [comment]
I'll merge this PR and afterwards we can re-check for the screenshots.

**Comment by ann0see** [comment]
Seems to be ok now.

**Comment by ann0see** [comment]
No... never mind. I'll copy them manually

**Comment by jujudusud** [comment]
@ignotus666 : if a screenshot is not translated, is the HTML file linked to the EN version automatically ?

**Comment by ignotus666** [comment]
Not that I know of.

**Comment by ann0see** [comment]
No. PT screenshots are totally missing. Also in english we have outdated ones (at least they don't show rc2).

**Comment by ann0see** [comment]
> if a screenshot is not translated, is the HTML file linked to the EN version automatically ?

I don't think polyglot (the language plug in we're using) supports this.

**Comment by ann0see** [comment]
@jujudusud Maybe we are lucky: https://github.com/untra/polyglot#configuration let me have a look at it. Probably we need a bit of restructuring then :-). But that's not too bad

**Comment by ann0see** [comment]
No. Unfortunately not. Probably we could somehow make it work with some logic in screenshot.html

**Comment by ignotus666** [comment]
Until a better solution is found or the screenshots are made, what about just creating a PT folder for the screenshots and copying the missing ones from EN? I know, not the best solution but it buys time until it gets sorted out, without having to edit all the links.

**Comment by jujudusud** [comment]
> No. PT screenshots are totally missing. Also in english we have outdated ones (at least they don't show rc2).

For PT, [I add a comment here](https://github.com/jamulussoftware/jamuluswebsite/issues/469#issuecomment-852277064)
Before your frenetic merges in changes, they were showing rc2 :-p , @gilgongo do you have your screenshots locally and can you delete the non used and squash those who are now in translate3_8_0 branch ?

[Edit] I also have to update, at the latest state of the interface, the FR screenshots !

**Comment by gilgongo** [comment]
I think we've established that it's just the main screen that isn't showing the new status lights. I don't have any images locally any more, I'm afraid.

**Comment by ann0see** [comment]
Taking screenshots is not a big deal ;-). I'm sure that we could take missing ones ourselves

**Pull request merged**